bin/README.md

```diff
@@ -0,0 +1,5 @@
+Excutable folder
+================
+
```

Comment:
When you are not going to edit the autogenerated files, better remove them. I usually remove this particular one, as how to execute the component is usually explained in the main Readme file.